rosdep/osx-homebrew.yaml

```diff
@@ -222,8 +222,6 @@ libflann-dev:
 libfltk-dev:
   osx:
     homebrew:
-      packages: [fltk]
-libftdi-dev:
```

Comment:
what did happen here?
seems that you should restore those lines.



CI output:

> 225:3 [key-duplicates] duplication of key "osx" in mapping

Reply:
Yes that looks like a typo. Will fix.